data/blog/2020-season-of-docs.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Google Season of Docs
+author:
+- erickzhao
+date: '2020-05-21'
+---
+
+Electron is proud to be participating in the second edition Google Season of Docs initiative, which pairs mentors from open source organizations with technical writers to improve project documentation.
+
+---
+
+# What is Season of Docs?
+
+![Season of Docs logo](https://developers.google.com/season-of-docs/images/logo/SeasonofDocs_Logo_SecondaryGrey_300ppi.png)
```

Comment:
Optional: Maybe host locally in case that image goes away.

Reply:
Good idea. Seems like past blog posts have used the GitHub CDN, so I've uploaded the image there instead.